app/models/form_submission_attempt.rb

```diff
@@ -25,15 +25,40 @@ class FormSubmissionAttempt < ApplicationRecord
 
     event :fail do
       after do
+        user_account_uuid = form_submission.user_account_id
+        form_type = form_submission.form_type
+        log_info = { form_submission_id:, benefits_intake_uuid: form_submission&.benefits_intake_uuid, form_type:,
+                     user_account_uuid: }
+        zsf_function = "#{form_type} form submission to Lighthouse"
         if should_send_simple_forms_email
-          Rails.logger.info({
-                              message: 'Preparing to send Form Submission Attempt error email',
-                              form_submission_id:,
-                              benefits_intake_uuid: form_submission.benefits_intake_uuid,
-                              form_type: form_submission.form_type
-                            })
+          Rails.logger.info('Preparing to send Form Submission Attempt error email', log_info)
           simple_forms_enqueue_result_email(:error)
+        elsif form_submission.form_type == CentralMail::SubmitForm4142Job::FORM4142_FORMSUBMISSION_TYPE
+          if Flipper.enabled?(CentralMail::SubmitForm4142Job::POLLED_FAILURE_EMAIL)
+            zsf_function = CentralMail::SubmitForm4142Job::FORM4142_DD_ZSF_FUNCTION
+            form526_submission_id = Form526Submission.find_by(saved_claim_id:).id
+            Rails.logger.info('Queuing Form526:Form4142 failure email to VaNotify',
+                              log_info.merge({ form526_submission_id: }))
+            jid = EVSS::DisabilityCompensationForm::Form4142DocumentUploadFailureEmail.perform_async(
+              form526_submission_id
+            )
+            Rails.logger.info('Queuing Form526:Form4142 failure email to VaNotify completed', log_info.merge({ jid: }))
+          else
+            Rails.logger.info(
+              'Would queue EVSS::DisabilityCompensationForm::Form4142DocumentUploadFailureEmail, but flipper is off.',
+              log_info
+            )
+          end
         end
+      rescue => e
+        cl = caller_locations.first
+        call_location = ZeroSilentFailures::Monitor::CallLocation.new(zsf_function, cl.path, cl.lineno)
+        ZeroSilentFailures::Monitor.new(Form526Submission::ZSF_DD_TAG_SERVICE)
```

Comment:
Eh this has `ZeroSilentFailure` in the name so maybe it is clear enough to use `ZSF` around it, disregard my other comment

Reply:
Oh I thought the opposite, since this has it fully spelled out (this is a class made by another team) that I should also spell it out fully everywhere.... ill mull on that